README.md

```diff
@@ -0,0 +1,13 @@
+# La Cabane Aux Etoiles
+
+## Install and run locally
+```
+npm install
+npm run dev
```

Comment:
Are you sure this is the correct command ?

Reply:
Yes totally 😄